app/src/main/java/com/example/metachef/RequestManager.java

```diff
@@ -47,9 +47,30 @@ public void onFailure(@NonNull Call<RandomRecipesResponse> call, @NonNull Throwa
         });
     }
 
-    public void getSearchRecipes(RecipeDetailsListener listener, List<String> id){
+    public void getSearchRecipes(SearchRecipesListener listener, String query){
         SearchRecipesCall searchRecipesCall = retrofit.create(SearchRecipesCall.class);
-        Call<SearchRecipesResponse> call = searchRecipesCall.callSearchRecipes(id, "a0b47258ef634097812d0213ca6217ea");
+        Call<SearchRecipesResponse> call = searchRecipesCall.callSearchRecipes(query, null, null,null, null,"a0b47258ef634097812d0213ca6217ea");
```

Comment:
Hide your API keys, usually you define this in a file thats hidden by your .gitignore
https://stackoverflow.com/questions/57355856/how-to-hide-api-url-and-api-key-from-apiresources-java

Reply:
Made these changes in (df2b500) commit